changelog.d/453.doc

```diff
@@ -0,0 +1 @@
+Be explicit that identifiers in the permissions config need to be wrapped in quotes, lest the angry yaml gods be enraged.
```

Comment:
```suggestion
Be explicit that identifiers in the permissions yaml config need to be wrapped in quotes, because they start with the characters @ and !.
```

Reply:
Aw, no fun! I suppose this is *slightly* more explicit.